src/main/core/Resources/config/services.yml

```diff
@@ -26,6 +26,9 @@ services:
             - method: setDispatcher
               arguments:
                 - '@Claroline\AppBundle\Event\StrictDispatcher'
+            - method: setRepository
+              arguments:
+                - '@Claroline\CoreBundle\Repository\User\UserRepository'
```

Comment:
Grab the repository from the `ObjectManager`. The `UserRepository` is a service because of a deprecated feature which needs DI.

Reply:
Done